tests/test_cli.py

```diff
@@ -874,16 +874,16 @@ def test_show_priority(tmpdir, runner, create):
     create(
         'test.ics',
         'SUMMARY:harhar\n'
-        'Priority:High\n'
+        'Priority:high\n'
```

Comment:
Priority here should be `1`. You should really use the `todo_factory` helper instead of `create`.